docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -213,7 +213,7 @@ You can change the density of the rows and the column header.
 
 ### Density prop
 
-The vertical density of the data grid can be set using the `density` prop. The `density` prop applies the values determined by the `rowHeight` and `headerHeight` props, if supplied. The user can override this setting with the toolbar density selector, if provided.
+To enable the density selector you need to add the `showToolbar` prop to the data grid. The vertical density of the data grid can be set using the `density` prop. The `density` prop applies the values determined by the `rowHeight` and `headerHeight` props, if supplied. The user can override this setting with the toolbar density selector, if provided.
```

Comment:
I think that we should isolate the sections. 


```suggestion
The vertical density of the data grid can be set using the `density` prop. The `density` prop applies the values determined by the `rowHeight` and `headerHeight` props, if supplied. The user can override this setting with the toolbar density selector, if provided.
```

Reply:
Done